esx_service/cli/vmdkops_admin_test.py

```diff
@@ -735,30 +735,56 @@ def test_tenant_access(self):
         # change total_volume size to 2GB
         volume_maxsize_in_MB = convert.convert_to_MB("1000MB")
         volume_totalsize_in_MB = convert.convert_to_MB("3GB")
-        error_info = auth_api._tenant_access_set(name=self.tenant1_name,
-                                                 datastore=self.datastore_name,
-                                                 allow_create="True",
-                                                 volume_maxsize_in_MB=volume_maxsize_in_MB,
-                                                 volume_totalsize_in_MB=volume_totalsize_in_MB
-                                             )
-        self.assertEqual(None, error_info)
 
-        error_info, privileges = auth_api._tenant_access_ls(self.tenant1_name)
-        self.assertEqual(None, error_info)
+        for val in ["False", "FALSE", "false", "Invalid"]:
```

Comment:
I don't follow the logic here: the first 3 values are valid input, but the last one is invalid. However, line 746 assume the returned error_info will be None, which is not correct for the last invalid input.